### ISSUE TITLE
AttributeError: 'ImportedAddressWallet' object has no attribute 'receiving_addresses'

### ISSUE THREAD

**Issue by ProfessorVenkman** [Electron-Cash/Electron-Cash#940]
Crash Report
============

This crash report was reported through the automatic crash reporting system 🤖

Traceback
--------------

```Python traceback
  File "site-packages\electron_cash-3.3.1cs-py3.5.egg\electroncash_gui\qt\main_window.py", line 1422, in request_password
  File "site-packages\electron_cash-3.3.1cs-py3.5.egg\electroncash_gui\qt\main_window.py", line 2185, in start_coinsplit
  File "site-packages\electron_cash-3.3.1cs-py3.5.egg\electroncash_gui\qt\coinsplit.py", line 25, in show_dialog
  File "site-packages\electron_cash-3.3.1cs-py3.5.egg\electroncash_gui\qt\coinsplit.py", line 54, in __init__
  File "site-packages\electron_cash-3.3.1cs-py3.5.egg\electroncash\wallet.py", line 400, in get_address_index

AttributeError: 'ImportedAddressWallet' object has no attribute 'receiving_addresses'
```

Reporter
------------

This issue was reported by 6 user(s):

| Electron Cash Version  | Python Version | Operating System  | Wallet Type  | Locale |
|---|---|---|---|---|
| 3.3.1CS  | 3.5.4 (v3.5.4:3f56838, Aug  8 2017, 02:07:06) [MSC v.1900 32 bit (Intel)] | Windows-10-10.0.17134 | imported_addr | hr_HR | 
| 3.3.1CS  | 3.5.4 (v3.5.4:3f56838, Aug  8 2017, 02:07:06) [MSC v.1900 32 bit (Intel)] | Windows-10-10.0.17134 | imported_addr | en_US | 
| 3.3.1CS  | 3.5.4 (v3.5.4:3f56838, Aug  8 2017, 02:07:06) [MSC v.1900 32 bit (Intel)] | Windows-8.1-6.3.9600 | imported_addr | de_DE | 
| 3.3.1CS  | 3.5.4 (v3.5.4:3f56838, Aug  8 2017, 02:07:06) [MSC v.1900 32 bit (Intel)] | Windows-7-6.1.7601-SP1 | imported_addr | en_US | 
| 3.3.1CS  | 3.5.4 (v3.5.4:3f56838, Aug  8 2017, 02:07:06) [MSC v.1900 32 bit (Intel)] | Windows-10-10.0.17134 | imported_addr | zh_CN | 
| 3.3.1CS  | 3.5.4 (v3.5.4:3f56838, Aug  8 2017, 02:07:06) [MSC v.1900 32 bit (Intel)] | Windows-10-10.0.17134 | imported_addr | zh_CN | 

Additional Information
------------------------

The reporting user(s) did not provide additional information.

**Comment by cculianu** [comment]
Paging @markblundeberg  :)

**Comment by markblundeberg** [comment]
fixed too in 3.3.2CS :-D

**Comment by ProfessorVenkman** [comment]
Good afternoon @markblundeberg,

I just received another crash report related to this issue. The crash occured on Electron Cash 3.3.4CS.
I'm not sure which versions of Electron Cash include the fix but this is the first report from anything
newer than 3.3.1CS since you closed the issue.

Could you please check if this issue really is resolved? Here is the traceback that I just collected:

```Python traceback
  File "site-packages\electron_cash-3.3.4cs-py3.5.egg\electroncash_gui\qt\address_list.py", line 190, in <lambda>
  File "site-packages\electron_cash-3.3.4cs-py3.5.egg\electroncash_gui\qt\main_window.py", line 1401, in request_password
  File "site-packages\electron_cash-3.3.4cs-py3.5.egg\electroncash_gui\qt\main_window.py", line 2191, in start_coinsplit_mul
  File "site-packages\electron_cash-3.3.4cs-py3.5.egg\electroncash_gui\qt\coinsplitmul.py", line 26, in show_dialog
  File "site-packages\electron_cash-3.3.4cs-py3.5.egg\electroncash_gui\qt\coinsplitmul.py", line 60, in __init__
  File "site-packages\electron_cash-3.3.4cs-py3.5.egg\electroncash\wallet.py", line 400, in get_address_index

AttributeError: 'ImportedAddressWallet' object has no attribute 'receiving_addresses'
```


~ _With robotic wishes_
